ext_data/community_models/README.md

```diff
@@ -0,0 +1,11 @@
+This is an example directory where you need to add all your `.mat` or `.json` model files 
+your community consists of. 
+
+For example, you may have a community of 2 models, *Escherichia coli* and *Helicobacter pylori*, 
+in that case you could get the [`e_coli_core.json`](http://bigg.ucsd.edu/models/e_coli_core) and 
+[`iIT341.json`](http://bigg.ucsd.edu/models/iIT341) files from the [BIGG database](http://bigg.ucsd.edu/models)
+and then run `dingo`. 
```

Comment:
could you please add an example of use `i.e.` a command from terminal that uses dingo to do some computations to e_coli and iIT341 community

Reply:
Could you have a look now?